console/src/components/Error/ErrorBoundary.tsx

```diff
@@ -60,9 +69,20 @@ class ErrorBoundary extends React.Component {
     );
   }
 
+  initialState = {
+    hasError: false,
+    info: null,
+    error: null,
+    type: '500',
+  };
+
+  resetState = () => {
+    this.setState({ ...this.initialState });
```

Comment:
I know it's old code, but could you change it to the code below as there's no need to copy here:
```suggestion
    this.setState(this.initialState);
```